params/config.go

```diff
@@ -111,15 +117,22 @@ type ChainConfig struct {
 	IstanbulBlock       *big.Int `json:"istanbulBlock,omitempty"`       // Istanbul switch block (nil = no fork, 0 = already on istanbul)
 	MuirGlacierBlock    *big.Int `json:"muirGlacierBlock,omitempty"`    // Eip-2384 (bomb delay) switch block (nil = no fork, 0 = already activated)
 
-	SubnetEVMTimestamp *big.Int `json:"subnetEVMTimestamp,omitempty"` // A placeholder for the latest avalanche forks (nil = no fork, 0 = already activated)
-
-	FeeConfig          commontype.FeeConfig `json:"feeConfig"`                    // Set the configuration for the dynamic fee algorithm
-	AllowFeeRecipients bool                 `json:"allowFeeRecipients,omitempty"` // Allows fees to be collected by block builders.
+	NetworkUpgrades              // Config for timestamps that enable avalanche network upgrades
+	PrecompileUpgrade            // Config for enabling precompiles from genesis
```

Comment:
On the fence of whether we should call this `PrecompileActivations`.

Nonetheless, I would make this plural to match `NetworkUpgrades`.

Reply:
This is a little nuanced since in the (genesis) chain config, multiple upgrades can be specified (one of each kind of precompile).
In `UpgradeConfig`, each `PrecompileUpgrade` is only allowed to specify one key, then we have an array of them which makes `PrecompileUpgrades`.

Definitely happy to rename whichever way is clearer.